homeassistant/components/airscape/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "domain": "airscape",
+    "name": "Airscape Whole House Fan",
+    "documentation": "https://www.home-assistant.io/components/fan.airscape",
```

Comment:
Remove the fan part in the url.

Reply:
All of the existing example that I have found for just platform extensions of a component reference documentation in this fashion.  
ie:
https://www.home-assistant.io/components/light.switch/
https://www.home-assistant.io/components/cover.mqtt/
https://www.home-assistant.io/components/binary_sensor.mysensonsors/

I couldn't find any documentation on what the proper formatting of the URL should be so I went with existing examples.

Reply:
Those examples are exceptions to the rule. Nowadays we use a single page per integration except for integrations that have many platforms with very long documentation.

See eg wemo:
https://www.home-assistant.io/components/wemo/

It has a fan platform.